<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd">
<suite name="Failed suite [Command line suite]" verbose="0">
  <test thread-count="5" name="Command line test(failed)" verbose="0">
    <classes>
      <class name="org.rntgroup.test.TrigonometricFunctionTest">
        <methods>
          <include name="unitTgTest" invocation-numbers="1 2 3 4"/>
          <include name="unitCtgTest" invocation-numbers="0 1 2"/>
          <include name="unitSinTest" invocation-numbers="1"/>
          <include name="tearDown"/>
          <include name="setUp"/>
          <include name="unitCosTest" invocation-numbers="0 1 2"/>
        </methods>
      </class> <!-- org.rntgroup.test.TrigonometricFunctionTest -->
      <class name="org.rntgroup.test.PowerFunctionTest">
        <methods>
          <include name="unitSqrtNegativeNumbersTest"/>
          <include name="tearDown"/>
          <include name="unitPowTest" invocation-numbers="2"/>
          <include name="setUp"/>
        </methods>
      </class> <!-- org.rntgroup.test.PowerFunctionTest -->
    </classes>
  </test> <!-- Command line test(failed) -->
</suite> <!-- Failed suite [Command line suite] -->
